app/presenters/wpcc/presenter.rb

```diff
@@ -16,6 +16,10 @@ def salary_frequency_options
       end
     end
 
+    def formatted_number(value, unit)
+      number_to_currency(value, unit: unit, precision: 2)
```

Comment:
Is the precision of 2 used in the majority of places? If it hasn't yet been a requirement then don't worry about changing it. But I can see a requirment where in some places the precision may only need to be `0`